cmake/Modules/KatanaBuildCommon.cmake

```diff
@@ -303,6 +304,12 @@ if (CMAKE_PROJECT_NAME STREQUAL PROJECT_NAME AND BUILD_TESTING)
   find_package(benchmark REQUIRED)
 endif ()
 
+# Instrument binaries if desired
+if (KATANA_ENABLE_COVERAGE)
+  set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -g -coverage")
```

Comment:
Should this also be set for C? I don't know if we have such code at the moment, but future proofing.

Reply:
We do not have those files at the moment; that said, I agree with you that we should include those flags.